### DISCUSSION TITLE
Accessing the value of a ListItem

### DISCUSSION THREAD

**Discussion by tmceld** [Textualize/textual#2259] [Q&A] [answered]
I am really sorry to use this platform for a 'how do i..?' question, but i am tearing my hair out here, and perhaps something could be updated in the (otherwise excellent) docs?

But how do I access the value of a ListItem ? 

I have 2 verticals, one has a list, when i click one, i want the second vertical to display that value (well it will have more functionality than that further down the line) but i can't access the value of the list item?  I looked in dictionary.py and code_broser.py and see that .value can be used there, but seemingly not with ListItem?  

I realise that I am probably fundamentally misunderstanding something key here, but would really appreciate some help.

If anyone would like to get bonus internet-points for this, i have put this on SO with a sloppy-code example: https://stackoverflow.com/questions/75986727/getting-the-value-of-a-listitem-listview-in-textualize-textual

Many thanks

**Comment by davep** [comment, marked as answer]
You may find [this previous discussion](https://github.com/Textualize/textual/discussions/1840) helpful.

**Reply by tmceld** [reply]
That was great, thank you.  I think that conceptually there is a lot i don't 'get' intuitively with textual, having only used python as a scripting language in the past and having no experience with front-end languages.  But this definately gets me on my way - much appreciated.

**Reply by davep** [reply]
Most welcome, glad it helped. Always [feel free to ask away here, or over in Discord if that's more your thing](https://textual.textualize.io/help/).
